courses/free-courses-en.md

```diff
@@ -1392,6 +1392,7 @@
 * [Python Tutorials](https://youtube.com/playlist?list=PLWKjhJtqVAbnqBxcdjVGgT3uVR10bzTEB) - freeCodeCamp.org
 * [SoloLearn](https://www.sololearn.com/Course/Python/)
 * [The Python Tutorial](https://docs.python.org/3/tutorial/)
+* [Think Python: Edition 1](https://greenteapress.com/thinkpython/thinkpython.pdf) - Allen downey (PDF)
```

Comment:
This is a book, not a free course. The book you mentioned already exists in the repository "free-programming-books-langs.md." Please check for duplicates before contributing next time

Reply:
My bad on the book part, the book that is already here is the second edition which focuses on Python 3 while this has 2, even stated by the publishers that depending on which you use, one of those books would be better. I will chance the course error.

> This is a book, not a free course. The book you mentioned already exists in the repository "free-programming-books-langs.md." Please check for duplicates before contributing next time